flatpak/org.mamedev.MAME.metainfo.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.mamedev.MAME</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0-or-later</project_license>
+  <name>MAME</name>
+  <summary>
+    MAME is a multi-purpose emulation framework.
+  </summary>
+  <description>
+    <p>MAME&apos;s purpose is to preserve decades of software history. As electronic technology continues to rush forward, MAME prevents this important &quot;vintage&quot; software from being lost and forgotten. This is achieved by documenting the hardware and how it functions. The source code to MAME serves as this documentation. The fact that the software is usable serves primarily to validate the accuracy of the documentation (how else can you prove that you have recreated the hardware faithfully?). Over time, MAME (originally stood for Multiple Arcade Machine Emulator) absorbed the sister-project MESS (Multi Emulator Super System), so MAME now documents a wide variety of (mostly vintage) computers, video game consoles and calculators, in addition to the arcade video games that were its initial focus.</p>
+  </description>
+  <categories>
+    <category>Game</category>
+    <category>ArcadeGame</category>
+    <category>Emulator</category>
+  </categories>
+  <url type="homepage">https://www.mamedev.org</url>
+  <launchable type="desktop-id">org.mamedev.MAME.desktop</launchable>
+  <releases>
+    <release version="0.245" date="2022-06-30"/>
+  </releases>
+  <provides>
+    <binary>mame</binary>
+  </provides>
+  <content_rating type="oars-1.0">
+    <content_attribute id="violence-cartoon">intense</content_attribute>
+    <content_attribute id="violence-fantasy">intense</content_attribute>
+    <content_attribute id="violence-realistic">intense</content_attribute>
+    <content_attribute id="violence-bloodshed">intense</content_attribute>
+    <content_attribute id="violence-sexual">intense</content_attribute>
+    <content_attribute id="drugs-alcohol">moderate</content_attribute>
+    <content_attribute id="drugs-narcotics">moderate</content_attribute>
+    <content_attribute id="drugs-tobacco">moderate</content_attribute>
+    <content_attribute id="sex-nudity">intense</content_attribute>
+    <content_attribute id="sex-themes">intense</content_attribute>
+    <content_attribute id="language-profanity">intense</content_attribute>
+    <content_attribute id="language-humor">intense</content_attribute>
+    <content_attribute id="language-discrimination">intense</content_attribute>
+    <content_attribute id="money-advertising">moderate</content_attribute>
+    <content_attribute id="money-gambling">moderate</content_attribute>
+  </content_rating>
```

Comment:
There’s nothing included with MAME that contains this stuff.  MAME only contains metadata (titles etc. in MAME and the software lists) – there might be a little risqué language and occasional profanity, but nothing serious.  I mean, do you put content ratings like this on video players because someone might use them to play snuff films?

Reply:
Right, I was thinking from a parent's point of view.  If a parent is consenting to their child to use the software they would want to know what they could potentially be using it for.  If it instead says "none" it could be misleading.  I was expecting to see some software controls around this, but I don't believe there are any, I don't think there's any view into this data

Reply:
Yeah, but my point is they could use a video player, image viewer, text editor, or web browser to view all sorts of inappropriate things, too.  And if you want to get into the content that MAME *can* be used for, there are plenty of games that are not legal to play in various jurisdictions, for example depictions of:
* Gratuitous sexual violence
* Sexual nudity as a reward, not related to narrative
* Sexual nudity depicting individuals who don’t clearly appear to be adults

If you want to rate MAME based on content that it *can* run, it needs the “refused classification” or equivalent.